Dockerfile

```diff
@@ -1,8 +1,11 @@
 ARG ARCH="amd64"
-ARG TAG="v3.13.3"
+ARG TAG="v3.19.1"
 ARG UBI_IMAGE=registry.access.redhat.com/ubi7/ubi-minimal:latest
 ARG GO_IMAGE=rancher/hardened-build-base:v1.15.8b5
+ARG CNI_PLUGINS_IMAGE=rancher/hardened-cni-plugins:v0.9.1-build20210414
+
 FROM ${UBI_IMAGE} as ubi
+FROM ${CNI_PLUGINS_IMAGE} as cni
```

Comment:
This may sounds persnickety but if we are aliasing it as `cni` I would prefer that the build arg was `CNI_IMAGE` which maps to how we are doing it for UBI.

Reply:
I don't know what persnickety is, but it's a good point :)